docs/docs/tutorials/using_dask.md

```diff
@@ -1,27 +1,28 @@
 ---
 id: using_dask
-title: Working with big data using Dask
-description: Introduction to Dask
+title: Work with big data using Dask
+description: Introduction to Dask with Nebari
 ---
 
-# Working with big data using Dask
+# Work with big data using Dask
 
 ## Introduction
 
 Working with large datasets can pose a few challenges - perhaps the most common is memory limitations on the
 user's machine.
 
-[Dask](https://docs.dask.org/en/stable/) is a flexible, Open-Source library for parallel computing in Python. Dask
-allows data scientists the ability able to scale analyses from a sample dataset to the full, large-scale dataset
-with almost no code changes. It's a powerful tool that can revolutionize how you do analytics!
+[Dask](https://www.dask.org/) is a flexible, open source library for parallel and distributed computing in Python.
+Dask allows data scientists the ability able to scale analyses from a sample dataset to the full, large-scale dataset
+with almost no code changes.
+It's a powerful tool that can revolutionize how you do analytics!
 
 ## Dask integration on Nebari
 
 Nebari uses [Dask Gateway][dask-gateway] to expose auto-scaling compute clusters automatically
 configured for the user, and it provides a secure way to managing Dask clusters.
```

Comment:
We should cross reference the advanced settings doc here https://github.com/nebari-dev/nebari-docs/pull/115/files#diff-7f41f42d3602fd87b565db4e582bb2ff06b2a1da146eab2e602d2594623c3c0e